src/test/java/com/iota/iri/conf/ConfigTest.java

```diff
@@ -207,6 +216,8 @@ public void testIniParsingTestnet() throws Exception {
                 .append("NUMBER_OF_KEYS_IN_A_MILESTONE = 3").append(System.lineSeparator())
                 .append("DONT_VALIDATE_TESTNET_MILESTONE_SIG = true").append(System.lineSeparator())
                 .append("TIPSELECTION_ALPHA = 1.1").append(System.lineSeparator())
+                //doesn't do anything
+                .append("REMOTE")
```

Comment:
I think this will be confusing - there is no real reason why `REMOTE` should not work in the config file - I would not call that "not backward compatible".

Reply:
The thing is that I don't know if it is correct that you have two ways to manipulate API_HOST...
I can totally imagine someone setting his API host to some IP. Later on he will google how to open his node to the world and add the REMOTE option. He won't notice the API_HOST becuase there are so many configs...

The same problem can also exist with flags, but it is less bad since you have to retype them.

Reply:
ok, I agree for now.
Can you open an issue about that then?

Reply:
I added a comment on https://github.com/iotaledger/iri/issues/931 regarding this problem.